css/_jitsi_popover.scss

```diff
@@ -8,98 +8,87 @@
     min-width: 100px;
     padding: 1px;
     text-align: left;
-    color: #333333;
-    background-color: #ffffff;
+    color: $popoverFontColor;
+    background-color: $popoverBg;
     background-clip: padding-box;
-    border: 1px solid #cccccc;
-    border: 1px solid rgba(0, 0, 0, 0.2);
-    border-radius: 6px;
+    border: 1px solid $popoverBg;
+    border-radius: 3px;
     /*-webkit-box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);*/
     /*box-shadow: 0 5px 10px rgba(0, 0, 0, 0.4);*/
     white-space: normal;
     margin-top: -10px;
     margin-bottom: 35px;
-}
 
-.jitsipopover.black
-{
-    background-color: rgba(0,0,0,0.8);
-    color: #ffffff;
-}
+    &__title {
+        text-align: left;
+        margin: 0;
+        padding: 0;
+    }
 
-.jitsipopover-content {
-    padding: 9px 14px;
-    font-size: 10pt;
-    white-space:pre-wrap;
-    text-align: center;
-}
+    &__content {
+        padding: em(5) em(10);
+        font-size: em(14);
+        white-space:pre-wrap;
+        text-align: center;
+    }
 
-.jitsipopover > .arrow,
-.jitsipopover > .arrow:after {
-    position: absolute;
-    display: block;
-    width: 0;
-    height: 0;
-    border-color: transparent;
-    border-style: solid;
-}
+    &__icon {
+        margin-right: 2px;
+    }
 
-.jitsipopover > .arrow {
-    border-width: 11px;
-    left: 50%;
-    margin-left: -11px;
-    border-bottom-width: 0;
-    border-top-color: #999999;
-    border-top-color: rgba(0, 0, 0, 0.25);
-    bottom: -11px;
-}
-.jitsipopover > .arrow:after {
-    border-width: 10px;
-    content: " ";
-    bottom: 1px;
-    margin-left: -10px;
-    border-bottom-width: 0;
-    border-top-color: #ffffff;
-}
+    &__green
```

Comment:
Yes, I renamed it to `connection-info__upload` (it's related to connection info but not to jitsi popover) and extracted color to variable